Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/server/CovidCertificateApiV1.kt

```diff
@@ -15,16 +15,24 @@ interface CovidCertificateApiV1 {
     @POST("/version/v1/publicKey")
     suspend fun sendPublicKey(
         @Body requestBody: PublicKeyUploadRequest
-    )
+    ): Response<Any>
```

Comment:
does it return anything ? would `Response<Unit>` work ?

Reply:
That's needs to be tested, what the server returns, Unit is likely fine